### DISCUSSION TITLE
Is this project dying?

### DISCUSSION THREAD

**Discussion by 0x00A1FA** [vuejs/language-tools#3473] [General]
Hi,

There have not been quality updates for two weeks now. Is this project dying?

Thanks in advance.

**Comment by yyx990803** [comment]
This project is now the Vue-specific part built on top of https://github.com/volarjs/volar.js - a lot of work is going on in the lower level parts of the architecture.

Also - asking questions like this just because **2 weeks** of inactivity is quite frankly absurd. Open source projects that are in working state doesn't suddenly "die" just because their maintainers are busy on something else for 2 weeks.

**Comment by so1ve** [comment]
#2509 #2351 You are quite rude. Contribute if you met problems. No more complain, especially you haven't made any contribution to the vue ecosystem.

**Comment by johnsoncodehk** [comment]
Like Evan said I'm doing a lot of related work, just not in this repo, there are a lot of updates in progress. 🤞
